lib/src/server/server.dart

```diff
@@ -103,18 +103,22 @@ class Server {
       _connections.add(connection);
       // TODO(jakobr): Set active state handlers, close connection after idle
       // timeout.
-      connection.incomingStreams.listen(serveStream, onError: (error) {
+      final handlers = <ServerHandler>[];
```

Comment:
This could be like a memory leak: There can be many thousands of requests being handled via the same http/2 connection. This list might accumulate all of them even if the actual handlers have finished a long time ago.

Reply:
Indeed, thanks!

I updated the code to avoid this list. PTAL